README.md

```diff
@@ -5,8 +5,7 @@ Pull requests are welcome.
 
 ## Books
 
-[2012: "A Guide to Kernel Exploitation: Attacking the Core" by Enrico Perla and Massimiliano Oldani](https://github.com/aozhimin/awesome-iOS-resource/blob/master/Books/A%20Guide%20to%20Kernel%20Exploitation%20Attacking%20the%20Core.pdf)
-
+2012: "A Guide to Kernel Exploitation: Attacking the Core" by Enrico Perla and Massimiliano Oldani
```

Comment:
Put [this link](http://8.oldhacker.org/docs/A%20Guide%20to%20Kernel%20Exploitation%20Attacking%20the%20Core(1).pdf) here instead.